src/main/java/dk/dbc/opennumberroll/OpennumberRollConnector.java

```diff
@@ -87,26 +81,26 @@ public OpennumberRollConnector(FailSafeHttpClient failSafeHttpClient, String bas
      * @param level              timings log level
      */
     public OpennumberRollConnector(FailSafeHttpClient failSafeHttpClient, String baseUrl, OpennumberRollConnector.TimingLogLevel level) {
-        this.failSafeHttpClient = InvariantUtil.checkNotNullOrThrow(
-                failSafeHttpClient, "failSafeHttpClient");
-        this.baseUrl = InvariantUtil.checkNotNullNotEmptyOrThrow(
-                baseUrl, "baseUrl");
+        if (failSafeHttpClient == null || baseUrl == null) {
```

Comment:
Det er set at en enum var NULL hvilket gav anledning til visse former for ged i programmet.

Reply:
Tilføjet check af level parameter